libs/core/langchain_core/tools/structured.py

```diff
@@ -16,6 +16,7 @@
 )
 
 from pydantic import Field, SkipValidation
+from pydantic._internal._model_construction import ModelMetaclass
```

Comment:
We need to do this without reaching into `pydantic._internal` which is a private API for pydnatic.

The simplest way to achieve this is by comparing the doc-string we get to the default doc-string for BaseModel.